test/test.xunit.execution/Sdk/Frameworks/Runners/XunitTestCollectionRunnerTests.cs

```diff
@@ -38,6 +38,58 @@ public class XunitTestCollectionRunnerTests
         Assert.True(fixtureUnderTest.Disposed);
     }
 
+    [Fact]
+    public static async void DisposeAndAsyncLifetimeShouldBeCalledInTheRightOrder()
```

Comment:
`async void` - not super Idiomatic, but just following the rest of the code

Reply:
It's idiomatic for xUnit. :wink: